### PR TITLE
docs: How to list datasets in a workspace

### LINKED ISSUE #3403: docs: HOWTO list datasets in a workspace
> > I think the only change we need here is to add a section here: https://docs.argilla.io/en/latest/getting_started/installation/configurations/workspace_management.html If you provide a code snippet I'm happy to do this.
> 
> @nataliaElv docs are updated. Please, take a look at those changes.

Looks ok to me! Although it would be good to explain how to list the datasets in a workspace, to make sure there's none, and delete them if there are any. I think this is more intuitive from the UI but not sure if we explain anywhere in the docs how to do this from Python.

_Originally posted by @nataliaElv in https://github.com/argilla-io/argilla/issues/3358#issuecomment-1633723316_

### PR DESCRIPTION
**Pull request by kursathalat**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Closes #3403 

**Type of change**

(Remember to title the PR according to the type of change)

- [ ] Documentation update

**How Has This Been Tested**

(Please describe the tests that you ran to verify your changes.)

- [x] `sphinx-autobuild` (read [Developer Documentation](https://docs.argilla.io/en/latest/community/developer_docs.html#building-the-documentation) for more details)

**Checklist**

- [x] I added relevant documentation
- [x] I followed the style guidelines of this project
- [x] I did a self-review of my code
- [x] I made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I filled out [the contributor form](https://tally.so/r/n9XrxK) (see text above)
- [ ] I have added relevant notes to the `CHANGELOG.md` file (See https://keepachangelog.com/)

### CODE REVIEW

**Comment by davidberenstein1957** [comment]
Hi, perhaps we would be better of with a separate database_management page? what do you think @nataliaElv

File: docs/_source/getting_started/installation/configurations/workspace_management.md [hunk 1]
```diff
@@ -189,3 +189,55 @@ workspace = rg.Workspace.from_name("new-workspace")
...
+
+:::{note}
+To work with the datasets on Python, you need to log in to Argilla with `rg.init()`.
+:::
+
+### List `Datasets`
+
+#### Python client
+
+You can list the datasets within a specific workspace with the `list()` function as seen below. To specify the workspace, you can use the `workspace` argument. Otherwise, it will list all the dataset in all workspaces.
```
Comment (sdiazlor) [inline docs/_source/getting_started/installation/configurations/workspace_management.md]: small typo -> all the dataset in

**Comment by davidberenstein1957** [comment]
Or at least rename the section to workspace and dataset management or something?

File: docs/_source/getting_started/installation/configurations/workspace_management.md [hunk 2]
```diff
@@ -189,3 +189,55 @@ workspace = rg.Workspace.from_name("new-workspace")
...
+
+#### Python client
+
+You can delete any dataset by pulling it from the server by `from_argilla()` and calling the `delete()` function.
+
+```python
+rg.FeedbackDataset.from_argilla("my_dataset", workspace="admin").delete()
+```
+
+### Further filtering for `Datasets`
```
Comment (davidberenstein1957) [inline docs/_source/getting_started/installation/configurations/workspace_management.md]: maybe this is a bit too much info?

**Comment by nataliaElv** [comment]
> Hi, perhaps we would be better of with a separate database_management page? what do you think @nataliaElv

You mean a datasets management page? Yes, it would give this content more visibility, but I guess that page could have a bit more content than this. We need to be careful about how that could impact other content around managing datasets already in the docs, like creating a dataset.

**Review by davidberenstein1957 — APPROVED** [review]

**Pull request merged**